bundle/src/main/java/io/wcm/caconfig/editor/EditorProperties.java

```diff
@@ -80,4 +80,29 @@ private EditorProperties() {
    */
   public static final String WIDGET_TYPE_TEXTAREA = "textarea";
 
+  /**
+   * Widget type to add a tagbrowser selection widget to a string parameter.
+   */
+  public static final String WIDGET_TYPE_TAGBROWSER = "tagbrowser";
+
+  /**
+   * With this additional property the root path for the tag browser widget can be set:
+   * The root path is passed as string value of the property.
+   */
+  public static final String PROPERTY_TAGBROWSER_ROOT_PATH = "tagbrowserRootPath";
+
+  /**
+   * With this additional property the root path for the tag browser widget can be set:
+   * If set to true, the current configuration context path is used as root path.
+   */
+  public static final String PROPERTY_TAGBROWSER_ROOT_PATH_CONTEXT = "tagbrowserRootPathContext";
```

Comment:
this property should be removed - it seems not to be used in this PR, and i see not use case for it for browsing tags.